server/routes/user.js

```diff
@@ -0,0 +1,35 @@
+var express = require("express");
```

Comment:
I think you should change the names of these routes to the plurals. That matches better with the examples we've done in class

Reply:
This has been changed to match the suggestion